core/src/main/resources/hudson/model/Job/configure_pt_BR.properties

```diff
@@ -25,3 +25,5 @@ name=Nome do {0}
 Description=Descri\u00e7\u00e3o
 Save=Salvar
 LOADING=CARREGANDO
+# Apply
+Apply=Aplicar
```

Comment:
That's an interesting one. In other systems I normally see "Concluir", but I guess in other parts we are probably using Aplicar. So +1

Reply:
ok :)